bpf/dns_tracker.h

```diff
@@ -0,0 +1,141 @@
+/*
+    light weight DNS tracker using trace points.
+*/
+
+#ifndef __DNS_TRACKER_H__
+#define __DNS_TRACKER_H__
+#include "utils.h"
+
+#define DNS_PORT        53
+#define DNS_QR_FLAG     0x80
```

Comment:
This is wrong. The QR flag is the top-most bit in a 16-bit value, so it should be `0x8000`. This value happens to work on a little-endian system because you're not doing ntohs() before the compare, but it'll break on big-endian...

Reply:
fixed